tests/p2p/test_header_request_object.py

```diff
@@ -13,6 +14,23 @@
 BLOCK_HASH = b'\x01' * 32
 
 
+class HeaderRequest(BaseHeaderRequest):
+    MAX_HEADERS_FETCH = 192
```

Comment:
I thought you couldn't overwrite an `@abstractmethod` by defining a class attribute in a subclass. Am I missing something?

Reply:
Brief testing shows this works as expected.  The parent class with the abstract property disallows instantiation.  The child class with the property set, allows instantiation and proper attribute access.